EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1230,273 +1230,273 @@ QUEST_LV_0300_20160310_001229	Even if you can't trust me you should trust Vaidot
+QUEST_LV_0300_20160310_001255	He is too old... Is he going to be alright...?
+QUEST_LV_0300_20160310_001256	You are not blocked by the black wall since you are the Revelator, as expected.
+QUEST_LV_0300_20160310_001257	The evil idol that was created by Dominikas in front of Pagarba Chapel...{nl}The black wall started to cover the monastery because of that idol.{nl}
+QUEST_LV_0300_20160310_001258	If we destroy that evil idol...{nl}There might be a good chance that the whole wall may disappear.
+QUEST_LV_0300_20160310_001259	I just don't know how I can destroy it...{nl}There is nothing I can think of at the moment.{nl}
+QUEST_LV_0300_20160310_001260	But, wait...{nl}I can sense the soul of the demons in you.{nl}
+QUEST_LV_0300_20160310_001261	That sphere you have...{nl}Is a demon's soul locked inside it?{nl}
+QUEST_LV_0300_20160310_001262	That's it.{nl}Maybe we can use this to destroy the evil idol.{nl}
+QUEST_LV_0300_20160310_001263	The idea is to purify the soul of the demons and use their power.{nl}Unfortunately, we lack the right amount of souls to destroy the idol.{nl}
+QUEST_LV_0300_20160310_001264	Can you fill the demon's soul a little more?{nl}As you already may know, defeating demons will automatically absorb their souls.
+QUEST_LV_0300_20160310_001265	The demons are strong so... I guess that their souls are strong as well.{nl}My sacred energy is not enough with this much power.
+QUEST_LV_0300_20160310_001266	We should somehow borrow the power of holy water...{nl}Like I told you before though, I have nothing at the moment.
+QUEST_LV_0300_20160310_001267	I am so sorry, but can you get me Holy Water from the Priest Master in Klaipeda?{nl}If we have enough holy water, we'll be able to purify the collapsed demon souls.
+QUEST_LV_0300_20160310_001268	Please use the statue of Goddess Vakarine near Viyme Fountain.{nl}You can quickly come back from Klaipeda.
+QUEST_LV_0300_20160310_001269	With this amount, you will be able to purify the Soul Sphere.{nl}Now, give me a moment.
+QUEST_LV_0300_20160310_001270	Since the power of a demon's soul is strong, the purified power is strong as well.{nl}This power has the capability to destroy the evil idol.{nl}
+QUEST_LV_0300_20160310_001271	Put the Soul Sphere's power into the purified evil idol.{nl}I am not sure if the idol is able to endure...
+QUEST_LV_0300_20160310_001272	Finally!{nl}Now, we can be freed from the black wall.{nl}
+QUEST_LV_0300_20160310_001273	It has been a very long time...{nl}I think my prayer to the goddess was not in vain...
+QUEST_LV_0300_20160310_001274	Now, we'll need to focus on the main culprit of the Tree of Truth sect, Dominikas...{nl}Everything will be over once we drive him out.{nl}
+QUEST_LV_0300_20160310_001275	Please allow me to lend your power a little more.{nl}Even if we have the methods to catch him, we cannot use our {nl}powers since the villagers are all possessed due to long torture...{nl}
```

Comment:
"Please lend me your power a little longer."

"we can't use our{nl} powers since the villagers have been possessed from being tortured so long...{nl}"